src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.SetThreadErrorMode.cs

```diff
@@ -9,6 +9,7 @@ internal partial class Interop
     internal partial class Kernel32
     {
         [DllImport(Libraries.Kernel32, SetLastError = true, ExactSpelling = true)]
```

Comment:
I do not see anything in `JIT_PollGC` that preserves the Win32 last error. Do we need to add `BEGIN/END_PRESERVE_LAST_ERROR` to it to make sure that the last error is not corrupted?

Reply:
Oh. That is not good. I would have thought that should have already been handled by that function. Let me step through it and see what I can find.

Reply:
Done.